test/resources/50-file-test-file.txt

```diff
@@ -0,0 +1,2 @@
+File message line 1
+File message line 2
```

Comment:
Is this file linked to this change?

Reply:
Err no - nothing to do with this node (hopefully). Will delete.